.github/workflows/e2e.yaml

```diff
@@ -4,7 +4,8 @@ on:
   pull_request:
   push:
     branches:
-      - "main"
+      - main
+      - v2
```

Comment:
Does it worth it to move the chart, update the bundle and then deprecate again the v2 chart?

Or it creates confusion to have it there?

Reply:
There are links in sites out of our control (like forums) and people are trying to look to the values file of the v3 in the `helm-charts` repository.

This effort is to clean the `helm-charts` repository as much as we can while making the release process and the maintenance burden the least painful.